library.json

```
@@ -1,31 +1,56 @@
 {
   "$schema": "https://raw.githubusercontent.com/platformio/platformio-core/develop/platformio/assets/schema/library.json",
-  "name": "LibraryTemplate",
-  "version": "1.0.0",
-  "description": "Templates for libraries compatible with the PlatformIO ecosystem",
-  "keywords": [
-    "template"
-  ],
+  "name": "PidController",
+  "version": "0.1.0",
+  "description": "A simple PID controller library for embedded systems",
+  "keywords": ["template"],
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/eagletrt/libpid-sw.git"
   },
   "authors": [
     {
-      "name": "Antonio Gelain",
-      "email": "antonio.gelain2@gmail.com",
+      "name": "Simone Ruffini",
+      "email": "",
+      "maintainer": false
+    },
+    {
+      "name": "Filippo Faccini",
+      "email": "",
+      "maintainer": false
+    },
+    {
+      "name": "Alessandro Conforti",
+      "email": "",
+      "maintainer": false
+    },
+    {
+      "name": "Valerio Cancemi",
+      "email": "valerio.cancemi04@gmail.com",
       "maintainer": true
     }
   ],
   "license": "AGPL-3.0-only",
   "frameworks": "*",
   "platforms": "*",
-  "headers": [],
-  "examples": [],
+  "dependencies": [
+    {
+      "name": "ArenaAllocator",
+      "version": "https://github.com/eagletrt/libarena-allocator-sw"
```

Specify dependency version as demonstrated [here](https://docs.platformio.org/en/latest/manifests/library-json/fields/dependencies.html):
```suggestion
      "version": "https://github.com/eagletrt/libarena-allocator-sw.git#v0.1.0"
```